### PR TITLE
fix: clarify test description for Use Rest Parameter with Function Parameter

### LINKED ISSUE #40707: Use Rest Parameter with Function Parameter Final Test is Unclear
While solving a problem on the free code camp of JavaScript ES6, I stuck to an error. Which was about the rest parameter.
The problem says to make a sum function which should take a rest parameter and return the sum of the parameters. I did the job but got an error about to use the args as the rest parameter, instead of using the args as the rest parameter. I called for help and reported the bug. You must change the last test, as it does not tells about the real issue or it must say to use the arrow function in order to create the sum funciton.

### PR DESCRIPTION
**Pull request by gikf**
Clarifies description of last test.

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #40707

### CODE REVIEW

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-the-rest-parameter-with-function-parameters.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ The result of `sum()` should be 0
 assert(sum() === 0);
 ```
 
-The `sum` function should use the `...` rest parameter on the `args` parameter.
+`sum` should be an arrow function and use the `...` rest parameter on the `args` parameter.
```
Comment (RandellDawson) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-the-rest-parameter-with-function-parameters.md:55]:
```suggestion
`sum` should be an arrow function and use rest parameters syntax (`...`) on the `args` parameter.
```

**Review by RandellDawson — DISMISSED** [review]
I believe it should be `rest parameter syntax` instead of `rest parameter` based on MDN.  I added also put `...` after `rest parameter syntax`.  This seems read better and be more consistent with how it appears in most articles.

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-the-rest-parameter-with-function-parameters.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ The result of `sum()` should be 0
 assert(sum() === 0);
 ```
 
-The `sum` function should use the `...` rest parameter on the `args` parameter.
+`sum` should be an arrow function and use rest parameters syntax (`...`) on the `args` parameter.
```
Comment (nhcarrigan) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-the-rest-parameter-with-function-parameters.md:55]:
```suggestion
`sum` should be an arrow function which uses rest parameter syntax (`...`) on the `args` parameter.
```

Total nitpick, but `rest parameter syntax` is not pluralised on MDN. Also, the wording feels a bit off, IMHO. How's this?

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-the-rest-parameter-with-function-parameters.md (hunk 2 above)
Comment (gikf) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-the-rest-parameter-with-function-parameters.md:55]: It indeed sounds more natural. I thought about usage of plural form as well when @RandellDawson requested the previous change. The way I explained it to myself is that while it's put in a single place, it accepts multiple parameters and pluralization comes mainly from the latter fact. Although if consider only the former then singular form makes more sense. I don't know which approach is better. On the MDN page of `spread syntax` an opposite of it, is even referred to as just `rest syntax`.

To summarize I'll be happy with whichever is deemed here more appropriate. What do you think?

Comment (nhcarrigan) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-the-rest-parameter-with-function-parameters.md:55]: image

I was basically following this grammatical approach. 🙂 

I'm not too set on one way or the other, though.

Comment (ShaunSHamilton) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-the-rest-parameter-with-function-parameters.md:55]: I agree the plural (_parameters_) sounds off. I would also argue the reason it should be singular is _it is **one**_ parameter. It just so happens to contain many parameters, but it itself is **a** parameter.

So, I would like the change suggested by Nicholas to be included.

**Review by naomi-lgbt — APPROVED** [review]
Looks good to me.

**Review by ShaunSHamilton — APPROVED** [review]
Congrats on your first pull request (PR)! 🎉

Thank you, for your contribution to the page! 👍
We are happy to accept these changes, and look forward to future contributions. 📝
